### DISCUSSION TITLE
Network environment type? e.g LAN WAN PAN SDN etc

### DISCUSSION THREAD

**Discussion by Eela566** [dannagle/PacketSender#267] [Q&A]
Kindly tell me the network environment type of packet sender because we don't have any information abou swtiches and routers on the badis of IPs and ports we are sending packets. In which network is t possible. e.g LAN WAN SDN etc... plz anyone answer me the in which network the simulation is done for Packet sender.

**Comment by dannagle** [comment]
Packet Sender operates at Layer 3 independent of any switch configuration. You don't need to do anything to your router other than ensure the target address is accessible. After that, Packet Sender is generating just UDP, TCP, and SSL/TLS (encrypted TCP). 

Hope that helps.
